test/runtime-test.js

```diff
@@ -947,7 +947,7 @@ describe('BEMHTML compiler/Runtime', function() {
       });
     });
 
-    it('should throw error with one apply', function() {
+    xit('should throw error with one apply', function() {
```

Comment:
The only question is: what shall I do with this cases? I think, as long as we introduce try/catch for block rendering it not valid anymore.